pkg/proxy/proxy.go

```diff
@@ -352,7 +352,10 @@ func Init(ctx context.Context, cfg *Config) (*Proxy, error) {
 	if len(cfg.ThinEvents) > 0 {
 		for _, event := range cfg.ThinEvents {
 			if _, found := validThinEvents[event]; !found {
-				cfg.Log.Infof("Warning: You're attempting to listen for \"%s\", which isn't a valid thin event\n", event)
+				// If not found in validThinEvents, check in validPreviewEvents
+				if _, foundInPreview := validPreviewEvents[event]; !foundInPreview {
+					cfg.Log.Infof("Warning: You're attempting to listen for \"%s\", which isn't a valid thin event or preview event\n", event)
+				}
```

Comment:
There's definitely a more elegant way to do this but this feels like the fastest solution for now